14/umbraco-commerce/release-notes/README.md

```diff
@@ -17,6 +17,16 @@ If you are upgrading to a new major version, check the breaking changes in the [
 
 This section contains the release notes for Umbraco Commerce 14 including all changes for this version.
 
+#### 14.0.0-alpha3 (24th June 2024)
+
+* Added Analytics section
+* Added Create Country modal to allow creating countries from ISO 3166 presets
+* Added license warning component throughout the commerce section
+* Updated Regions workspace app is now hidden until the country is persisted
```

Comment:
```suggestion
* Update: The Regions workspace app is now hidden until the country is persisted
```

Reply:
Maybe `Updated Regions workspace app to now be hidden until the country is persisted`

Just a little more consistent with all the other items 👍

Reply:
Makes sense! I'll make the changes.